nextest-runner/default-config.toml

```diff
@@ -12,6 +12,10 @@ dir = "target/nextest"
 # "retries" defines the number of times a test should be retried. If set to a
 # non-zero value, tests that succeed on a subsequent attempt will be marked as
 # non-flaky. Can be overridden through the `--retries` option.
+# Examples
+# * retries = 3
+# * retries = { backoff = fixed, count = 2, delay: "1s" }
+# * retries = { backoff = exponential, count = 10, delay: "1s", jitter: true, max_delay: "10s" }
```

Comment:
This needs to be in the TOML format:

`backoff = "fixed", delay = "1s", jitter = true`, etc.